library/service.go

```diff
@@ -471,3 +476,116 @@ func (s *Service) String() string {
 		s.GetStatus(),
 	)
 }
+
+// ServiceFromContainer converts the pipeline
+// Container type to a library Service type.
+//
+// nolint: funlen // ignore function length due to comments and conditionals
```

Comment:
is there anyway to configure our linter to ignore comments for funlen?

Reply:
Excellent question 👍 

Unfortunately, I'm not seeing a way to make this possible although I think it would be a great feature!

The `golangci-lint` docs don't show anything AFAICT:

* [list of linters](https://golangci-lint.run/usage/linters/)
* [configuration of linters](https://golangci-lint.run/usage/configuration/)

Here is the official repo for that linter which doesn't give many docs:

https://github.com/ultraware/funlen